test/performance/specs/post-editor.spec.js

```diff
@@ -149,6 +150,30 @@ test.describe( 'Post Editor Performance', () => {
 		} );
 	} );
 
+	test.describe( 'Typing with Top Toolbar', () => {
+		let draftId = null;
+
+		test( 'Setup the test post', async ( { admin, perfUtils, editor } ) => {
+			await admin.createNewPost();
+			await editor.setIsFixedToolbar( true );
```

Comment:
This is a global, persistent setting so it needs to be reverted after the test or it will affect other metrics. Let's also move this line to the `Run the test` scope. See [this PR](https://github.com/WordPress/gutenberg/pull/56780) as an example on how to handle it!